src/commands.cc

```diff
@@ -1107,7 +1108,12 @@ KeymapMode parse_keymap_mode(const String& str)
     if (prefix_match("user", str))   return KeymapMode::User;
     if (prefix_match("object", str)) return KeymapMode::Object;
 
-    throw runtime_error(format("unknown keymap mode '{}'", str));
+    auto it = std::find(user_modes.begin(), user_modes.end(), str);
```

Comment:
use the range based find wrapper provided by ranges.hh: `find(user_modes, str)` (it will actually use std::find, but does the iterator boilerplate for you).